### DISCUSSION TITLE
Activities with bluetooth

### DISCUSSION THREAD

**Discussion by nilmarpublio** [frontendbr/forum#957] [Perguntas e Respostas]
Hello people,
I ask your help in helping me solve a situation.
I have an application that should communicate with bluetooth.
So far so good.
What happens is that I read the message of a place via bluetooth,  depending on the content of this message I forward it to up to 3 different forms in other activities also via bluetooth.
What happens is that the bluetooth socket drops when I exit the activity I sent the data.
Can anyone help me with this ?

**Comment by leandroruel** [comment]
err....i think you need ask it in a proper place, like stackoverflow, you will get better answers.
